test/integration/targets/timezone/tasks/main.yml

```diff
@@ -1,15 +1,21 @@
-- name: set timezone to Etc/UTC
-  timezone:
-    name: Etc/UTC
+- block:
+    - name: Set timezone to Etc/UTC
```

Comment:
This task should be outside the block. That will avoid potential errors if the task were to fail and the always block tried to restore a previous timezone which hadn't yet been determined.